### ISSUE TITLE
Implement option to specify whether leaderboard should be based on total points acquired, or current balance

### ISSUE THREAD

**Issue by JobDoesburg** [KiOui/mentorpunten#93]
Maybe I acquire 100 points during a tournament, but in the store I buy things for 80 points. 
Should I be displayed in the leader board with 20 points (because I also spent points)? Or should I still be leading with 100 points (because that's still what I earned). 

This choice depends on your view on the tournament (whether the store is an actual game element, or a reward store). I think this behavior should be user-selected for each tournament (or store)

**Comment by KiOui** [comment]
Yes. Right now I implemented this rather quickly with a different account that keeps track of the "coins" that are spendable. If a tournament uses both spendable coins and points we could also use one account. An idea could be to only filter on positive transactions and let that be the score total. The reason why I did it this way is because we also want to be able to deduct points so we should keep that in mind as well.
